### DISCUSSION TITLE
Request for information: question

### DISCUSSION THREAD

**Discussion by cleanev** [teslamate-org/teslamate#2270] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

This is a request for information about size of database and whether there is any need/possibility to shrink database size.

I installed teslamate in July 2020 (Model Y), since then teslamate database has grown to 680MB, while not outrageously large, my other car where I installed teslamate (Model 3) in May 2020 has a size of just 281MB. 

This makes me wonder why one is 3 times the other? 

Other points 
- 3 is driven around 10500 miles during last 20 months, charged 165 times
- Y is driven around 21000 miles during last 18 months, charged 301 times

nothing 

### Expected Behavior

Request for information

### Steps To Reproduce

Request for information

### Relevant log output

```shell
Request for information
```

### Screenshots

N/A

### Additional data

N/A

### Type of installation

Docker

### Version

V1.25

**Comment by manuelkamp** [comment]
You have double the drives and double the charges on the Y - why do you think it should have only as much data stored as the db for your M3?
